app/assets/stylesheets/custom.css.scss

```diff
@@ -74,3 +74,13 @@ body {
 .create-item-form {
   width: 80%;
 }
+
+.item-create {
+  background: rgba(43,58,223,0.2);
+  width: 500px;
```

Comment:
Properties should be ordered background, border, border-color, border-style, margin-left, margin-right, width